### PR TITLE
new api

### PR DESCRIPTION
**Pull request by guschmue**

### CODE REVIEW

File: tf2onnx/convert.py [hunk 1]
```diff
@@ -193,56 +228,230 @@ def main():
...
+            inputs_as_nchw=inputs_as_nchw,
+            large_model=large_model,
+            tensors_to_rename=tensors_to_rename,
+            initialized_tables=initialized_tables,
+            output_path=output_path)
+
+        return model_proto, external_tensor_storage
+
+
+def from_graph(graph_def, name=None, input_names=None, output_names=None, opset=None, custom_ops=None,
```
Comment (TomWildenhain-Microsoft) [inline tf2onnx/convert.py:397]:
```suggestion
def from_graph_def(graph_def, name=None, input_names=None, output_names=None, opset=None, custom_ops=None,
```

The input is a graph_def not a tf graph

File: tf2onnx/convert.py [hunk 2]
```diff
@@ -193,56 +228,230 @@ def main():
...
+               custom_op_handlers=None, custom_rewriter=None, inputs_as_nchw=None, extra_opset=None,
+               shape_override=None, target=None, large_model=False, tensors_to_rename=None, output_path=None):
+    """Returns a ONNX model_proto for a tensorflow graphdef.
+
+    Args:
+        graphdef: the graphdef we want to convert
```
Comment (TomWildenhain-Microsoft) [inline tf2onnx/convert.py:403]:
```suggestion
        graph_def: the graphdef we want to convert
```

File: tf2onnx/tf_loader.py [hunk 3]
```diff
@@ -267,9 +270,9 @@ def _from_saved_model_v1(sess, model_path, input_names, output_names, tag, signa
             for _, input_tensor in inputs_tensor_info.items():
                 if input_tensor.name not in input_names:
                     input_names.append(input_tensor.name)
+    tensors_to_rename = {}
     if output_names is None:
         output_names = []
-        tensors_to_rename = {}
```
Comment (TomWildenhain-Microsoft) [inline tf2onnx/tf_loader.py:272]: Oops.  My bad.

**Review by TomWildenhain-Microsoft — APPROVED** [review]

File: tf2onnx/convert.py (hunk 1 above)
Comment (guschmue) [reply tf2onnx/convert.py:397]: done

**Pull request merged**